src/backend/opengl/CMakeLists.txt

```diff
@@ -186,12 +195,18 @@ TARGET_LINK_LIBRARIES(forge
     PRIVATE ${FREETYPE_LIBRARIES}
     PRIVATE ${WTK_LIBRARIES}
     PRIVATE ${OPENGL_gl_LIBRARY}
-    PRIVATE ${GLBINDING_LIBRARIES}
     PRIVATE ${FREEIMAGE_LIBRARY}
     PRIVATE ${X11_LIBS}
     )
 
-ADD_DEPENDENCIES(forge ${glsl_shader_targets} glbinding)
+ADD_DEPENDENCIES(forge ${glsl_shader_targets})
+IF(USE_SYSTEM_GLBINDING)
+    TARGET_LINK_LIBRARIES(forge PRIVATE glbinding::glbinding)
```

Comment:
@pavanky This is how the new style CMake IMPORTED targets work. CMake recommends the usage of a namespace for client-side discovery, hence the `glbinding::` prefix.